scripts/docgen.lua

```diff
@@ -224,7 +224,7 @@ local function generate_readme(template_file, params)
   local input_template = readfile(template_file)
   local readme_data = template(input_template, params)
 
-  local writer = io.open("README.md", "w")
+  local writer = io.open("GENERATED_README.md", "w")
```

Comment:
It doesn't matter to the user whether it is GENERATED or not, so why not just use "CONFIG.md"?

Reply:
So, I agree with you. The main reason I put GENERATED was because people keep filing PRs in which they've edited README.md, I'll update the PR.